packages/solecs/src/Ownable.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { Ownable as SolidStateOwnable } from "@solidstate/contracts/access/ownable/Ownable.sol";
+import { OwnableStorage } from "@solidstate/contracts/access/ownable/OwnableStorage.sol";
+
+/**
+ * IERC173 implementation
+ */
+contract Ownable is SolidStateOwnable {
+  constructor() {
+    // Initialize owner (SolidState has no constructors)
+    OwnableStorage.layout().owner = msg.sender;
```

Comment:
Because it's set in just 1 place and having an explicit method for it seemed unnecessary.
The method also emits an event.
Which is important, because after checking EIP-173, the event should be emitted on creation too.
Since it's merged I'll make a fix PR to use _setOwner, @holic thanks for noticing

Reply:
> Because it's set in just 1 place and having an explicit method for it seemed unnecessary.

By that I mean, _setOwner didn't exist in the previous version (I made the PR 2 months ago and recently updated solidstate), and when updating I didn't think to check for a method

Reply:
https://github.com/latticexyz/mud/pull/265#issuecomment-1372451237

and this comment is about another `setOwner` which was removed, it didn't emit an event